dev/com.ibm.ws.session.db/resources/com/ibm/ws/session/db/resources/WASSession.nlsprops

```diff
@@ -156,9 +156,9 @@ DatabaseHashMap.selectNoUpdateError.explanation=An exception has been detected w
 DatabaseHashMap.selectNoUpdateError.useraction=If a SQLException has occurred then refer to the appropriate database documentation for your environment. Also, ensure that you have properly configured a data source for the Session Manager.
 
 # -------------------------------------------------------------------------------------------------
-DatabaseHashMapMR.db2LongVarCharErr=SESN8558E: An attempt was made to write more than 2M to the large column.
+DatabaseHashMapMR.db2LongVarCharErr=SESN8558E: An attempt was made to write more than row size limit to the database column.
 DatabaseHashMapMR.db2LongVarCharErr.explanation=The session data may be too large for the database column.
-DatabaseHashMapMR.db2LongVarCharErr.useraction=Reduce the size of this attribute before attempting to store it in the session.
+DatabaseHashMapMR.db2LongVarCharErr.useraction=Reduce the size of this attribute before attempting to store it in the session or increase the rowSizeLimit property.
```

Comment:
change
or increase the rowSizeLimit property.
to
or increase the value of the rowSizeLimit property.

Reply:
Thank you for the quick reviews.  I have updated the messages.